src/game/game.controller.ts

```diff
@@ -0,0 +1,131 @@
+import { Request, Response } from 'express';
+import { instanceToPlain, plainToInstance } from 'class-transformer';
+import { validate } from 'class-validator';
+import { Controller } from '../common';
+import { GameService, AddGameDto, AddLevelDto } from '.';
+import { StatusCodes, ReasonPhrases } from 'http-status-codes';
+
+export class GameController extends Controller {
+    constructor() {
+        super('/game');
+
+        const router = this.getRouter();
+
+        router.get('/', this.getAllGames);
+        router.post('/', this.addGame);
+        router.get('/:id', this.getGame);
+        router.delete('/:id', this.removeGame);
+        router.post('/:id/experience', this.addExperienceLevel);
+        router.delete('/:id/experience/:lvl_id', this.removeExperienceLevel);
+    }
+
+    async getAllGames(req: Request, res: Response) {
+        return res.send(await GameService.getAllGames());
+    }
+
+    async getGame(req: Request, res: Response) {
+        const id = req.params.id;
+
+        try {
```

Comment:
remove the try catch block
no need to catch errors here
i already did this globally